### DISCUSSION TITLE
integração com metheora e IBGE

### DISCUSSION THREAD

**Discussion by pereira299** [BrasilAPI/BrasilAPI#211] [Show and tell]
Além das integrações já realizadas conheço duas APIs que podem contribuir de forma relevante nesse projeto:
- A  API de localidades do IBGE que considero bastante completa com informações de mesorregiões, microrregião, distritos e subdistritos. Link abaixo
https://servicodados.ibge.gov.br/api/docs/localidades?versao=1

- Também há a API de endereços da metheora, Não possui tantos detalhes e algumas informações ainda estão faltando (como por exemplo nomes de bairros), mas acredito que mesmo assim possa contribuir para o projeto como redundancia para garantir que os dados estão corretos. Link abaixo
http://enderecos.metheora.com/help

**Comment by enieber** [comment]
@pereira299 será que não seria interessante fazer similar o que o  @juniorpb  e o @filipedeschamps fez criando o CEP promisse e o Cidades promisse? assim o Brasil API só conectaria com essa apis e elas escolhem o melhor provedor de dados.

**Comment by pereira299** [comment]
@enieber Seria uma boa alternativa... Acredito que seria bom deixar o Brasil API pronto para se conectar com o IBGE ao menos, visto que possui dados bastante interessantes, como distritos e microrregiões. Assim, caso o usuario queira dados como distritos de uma cidade, por exemplo, o IBGE seria escolhido como provedor.

**Comment by Caaddss** [comment]
Issue Relacionada #67
